src/app.js

```diff
@@ -694,7 +707,9 @@ function run () {
     app: self.event,
     udapp: udapp.event
   }
-  var righthandPanel = new RighthandPanel(document.body, rhpAPI, rhpEvents, {}) // eslint-disable-line
+  var righthandPanel = new RighthandPanel(rhpAPI, rhpEvents, {}) // eslint-disable-line
+  self._view.rightpanel.replaceWith(righthandPanel.render())
```

Comment:
... I would rather make the `rightPanel` the container of righthandpanel instead of replacing